protocol/v2/ssv/validator/validator.go

```diff
@@ -65,9 +58,19 @@ func NewValidator(pctx context.Context, options Options) *Validator {
 		Storage:     options.Storage,
 		Share:       options.SSVShare,
 		Signer:      options.Signer,
-		Queues:      queues,
+		Queues:      make(map[spectypes.BeaconRole]msgqueue.MsgQueue), // populate below
 		state:       uint32(NotStarted),
 	}
+
+	indexers := msgqueue.WithIndexers(msgqueue.SignedMsgIndexer(), msgqueue.DecidedMsgIndexer(), msgqueue.SignedPostConsensusMsgIndexer(), msgqueue.EventMsgMsgIndexer())
+	for _, dutyRunner := range options.DutyRunners {
+		// set timeout F
+		dutyRunner.SetTimeoutF(v.onTimeout)
```

Comment:
@nivBlox Lets move the SetTimoutF to base runner rather then on each

Reply:
done